Bootable JAR 3.0 announcement

jfdenise opened the pull request:
@bstansberry @jamezp , This the Bootable JAR 3.0 announcement. The date has been set to tomorrow (15th of january).
Thank-you.

bstansberry commented on _posts/2021-01-15-bootable-jar-3.0.Final-Released.adoc:
```diff
@@ -0,0 +1,84 @@
+The version 3.0 of the link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/[WildFly Bootable JAR Maven plugin] has been released.
+
+For people who are not familiar with WildFly Bootable JAR, I strongly recommend that you read 
+this link:https://www.wildfly.org/news/2020/10/19/bootable-jar-2.0-released/[blog post] that covers it in details. 
```
s/details/detail

bstansberry commented on _posts/2021-01-15-bootable-jar-3.0.Final-Released.adoc:
```diff
@@ -0,0 +1,84 @@
+## Keycloak Galleon feature-pack
+
+The link:https://repo1.maven.org/maven2/org/keycloak/keycloak-adapter-galleon-pack/[Keycloak Galleon feature-pack] provides 
+an OIDC client adapter by the mean of a Galleon layer.
```
s/mean/means

bstansberry commented on _posts/2021-01-15-bootable-jar-3.0.Final-Released.adoc:
```diff
@@ -0,0 +1,84 @@
+The WildFly feature-pack is accessible from the Galleon `community-universe`. Retrieving the WildFly feature-pack from this universe
+allows us to not specify any version and get the latest one. 
+
+Keycloak Galleon feature-pack is not registered inside an universe, you need to specify its Maven coordinates.
```
s/Keycloak/Since the Keycloak/g

bstansberry commented on _posts/2021-01-15-bootable-jar-3.0.Final-Released.adoc:
```diff
@@ -0,0 +1,84 @@
+
+Keycloak Galleon feature-pack is not registered inside an universe, you need to specify its Maven coordinates.
+
+When building a Bootable JAR, the plugin retrieves the feature-packs from Maven repositories and provision a server based on the set of Galleon layers.
```
s/provision/provisions

bstansberry commented on _posts/2021-01-15-bootable-jar-3.0.Final-Released.adoc:
```diff
@@ -0,0 +1,84 @@
+
+## To conclude
+
+You have an interesting use-case, simple enough to be reduced to a simple example, and it is not covered 
```
s/You/If you/g

bstansberry reviewed (changes requested):
@jfdenise This looks fine. I have some minor, optional suggestions. I'll be sure to merge this tomorrow.

jfdenise commented:
@bstansberry , thank-you. I fixed your findings, we have released a 3.0.2 to address an issue on Windows, I have updated the doc to rely on this version. In addition I have added a small chapter to introduce the work that is being done with ODO+dev-watch on Openshift.

bstansberry commented:
Thanks @jfdenise
